### PR TITLE
Link to v2 Kubernetes instructions from platform docs

### PR DESCRIPTION
**Pull request by pierwill**
Work toward #2575

### CODE REVIEW

File: content/platform/integrations/kubernetes.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ stack) with Kubernetes:
   - [How is the InfluxData Platform (TICK) different from Prometheus?](#how-is-the-influxdata-platform-tick-different-from-prometheus)
   - [Should I run InfluxDB in Kubernetes?](#should-i-run-influxdb-in-kubernetes)
 
+## Quick start with minikube
+
+To start experimenting right away with the latest InfluxDB and Kubernetes, visit [Get started with InfluxDB](https://v2.docs.influxdata.com/v2.0/get-started/) and click **Kubernetes**.
+
```
Comment (sanderson) [inline content/platform/integrations/kubernetes.md]: We need to make sure it's clear that this method is only for InfluxDB v2. Maybe add it into the heading, link text, etc. 

This would be very confusing if somebody is in the v1.x docs looking for ways to get started with InfluxDB and Kubernetes and this link takes them to 2.0 without indication that it's only meant for 2.0. Many 1.x users are not ready to make the jump to 2.0, even with k8s.

Comment (gunnaraasen) [reply content/platform/integrations/kubernetes.md]: Agreed, we need to make it clear what is v1 or v2. I think it would probably be best to make this a blurb at the top informing users that there's a different set of k8s resources for v2 and this content only applies to v1.

**Comment by pierwill** [comment]
Pushed another version. How does this look?

File: content/platform/integrations/kubernetes.md [hunk 2]
```diff
@@ -12,8 +12,13 @@ has become a popular way to deploy and manage containers across multiple servers
 
 ![InfluxDB Kubernetes Logos](/img/platform/flux-kube.png)
 
-There are several ways use the InfluxData Platform (also known as the TICK
-stack) with Kubernetes:
+{{% note %}}
+This page is about using Kubernetes with the TICK stack and 1.x versions of InfluxDB.
+
+To start experimenting with InfluxDB version 2 and Kubernetes, visit [Get started with InfluxDB](https://v2.docs.influxdata.com/v2.0/get-started/) and click **Kubernetes**.
```
Comment (gunnaraasen) [inline content/platform/integrations/kubernetes.md]: "click Kubernetes" reads weird to me. How about this wording?

> ...go to the **Kubernetes** section of the [Get Started page for InfluxDB v2](https://v2.docs.influxdata.com/v2.0/get-started/).

Comment (sanderson) [inline content/platform/integrations/kubernetes.md]: ~InfluxDB version 2~ -> InfluxDB 2.0
